From b7dbef1ab135ffc2c04c759fce3a3311b4975e44 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 17 Feb 2015 21:41:00 -0500
Subject: [PATCH 094/315] session: plug memory leak

gdm_session_get_session_id erroneously strdup's it's return value.

https://bugzilla.gnome.org/show_bug.cgi?id=744764
---
 daemon/gdm-session.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gdm-session.c b/daemon/gdm-session.c
index 477ccb23..23501fbe 100644
--- a/daemon/gdm-session.c
+++ b/daemon/gdm-session.c
@@ -2815,7 +2815,7 @@ gdm_session_get_session_id (GdmSession *self)
                 return NULL;
         }
 
-        return g_strdup (conversation->session_id);
+        return conversation->session_id;
 }
 
 static char *
-- 
2.15.1

